sdk/src/trace/batch_span_processor.cc

```diff
@@ -206,6 +206,10 @@ void BatchSpanProcessor::Export()
       NotifyCompletion(notify_force_flush, exporter_, synchronization_data_);
       break;
     }
+
+    //reserves the space for the number of records to be exported
+    spans_arr.reserve(num_records_to_export);
+    
```

Comment:
```suggestion
    // reserves the space for the number of records to be exported
    spans_arr.reserve(num_records_to_export);

```

This should fix the CI failure on formatting.